### DISCUSSION TITLE
Where am I supposed to put `--uc-newTab-wallpaper` value?

### DISCUSSION THREAD

**Discussion by seaque** [black7375/Firefox-UI-Fix#822] [Q&A]
>user_pref("userContent.newTab.background_image",            true); // Need wallpaper image --uc-newTab-wallpaper: url("../icons/background_image.png");

I tried this in **userChrome.css** but wallpaper isn't changing. 

`:root { --uc-newTab-wallpaper:url("linktomyimage"); }`

**Comment by black7375** [comment]
You can add `:root { --uc-newTab-wallpaper:url("linktomyimage"); }` to `userConent.css`.
It's not clear what this means. I'll update it.
